### ISSUE TITLE
While applying Orientation newly created or multiple projects are not visible in Projects screen: A11y_.NET 9 template_Hambureger-Projects Options_Orientation.

### ISSUE THREAD

**Issue by keerthiduvvuri** [dotnet/maui#26917]
### GitHub Tags:
#A11yTCS; #A11ySev1; #Android; #WCAG1.3.4; #SH_.NET9template_Android_Dec2024; #Talkback; #.NET Core; #A11yMAS; #Orientation; #A11yWCAG2.1; #Device; #AINotInscope; 

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
Android Version: 14
Talkback version: 15.1

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Swipe till Hamburger menu and double tap on it, Projects options will screen will open.
4. Swipe till New button and double tap on it, New project screen will appear.
5. Swipe till category section and choose a category from the list of category.
6. Swipe till tags section and choose a tag from the list of tags or We can select the task from the existing task list.
8. Swipe till created project and Turn on Orientation mode and Observe that While applying Orientation newly created or multiple projects are not visible in Projects screen.

### Actual Result:
.While applying Orientation newly created or multiple projects are not visible in Projects screen

### Expected Result:
While applying Orientation newly created project or multiple projects should be visible in Projects screen, so that users wont miss the data.

### User Impact:
Users with dexterity impairments will face difficulty that newly created or multiple projects wont be visible in landscape mode, data is missing to users.

### Attachment:

https://github.com/user-attachments/assets/351a500b-67dd-4405-b0a5-847d10e38081

**Comment by InduPriya1805** [comment]
rev:inse

**Comment by merriemcgaw** [comment]
@tj-devel709 @PureWeen  does this just need to be validated by the CAI testers?

**Comment by keerthiduvvuri** [comment]
Verified in latest environment and Issue is Fixed, Hence closing the bug.
Adding the attachment for reference.

[26917_Closing Attachment.zip](https://github.com/user-attachments/files/21428116/26917_Closing.Attachment.zip)
